test_scripts/API/Expanded_proprietary_data_exchange/007_System_onSystemRequest_without_requestSubType_in_ptu.lua

```diff
@@ -0,0 +1,66 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0083-Expandable-design-for-proprietary-data-exchange.md
+-- User story:TBD
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1. PTU is performed without requestSubType and with requestType
+-- 2. SDL receives SystemRequest and onSystemRequest with requestSubType value
+-- SDL does:
+-- 1. send OnAppPermissionChanged without requestSubType and with requestType parameters to HMI during update
+-- 2. process SystemRequest request successful and resend onSystemRequest to mobile application
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Expanded_proprietary_data_exchange/commonDataExchange')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local usedFile = "./files/action.png"
+local params = {
+  requestType = "OEM_SPECIFIC",
+  requestSubType = "someValue",
+  fileName = "action.png"
+}
+
+--[[ Local Functions ]]
+local function ptuFuncRPC(tbl)
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].RequestType = { "PROPRIETARY", "OEM_SPECIFIC" }
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].RequestSubType = nil
+end
+
+local function policyUpdate()
+  common.policyTableUpdate(ptuFuncRPC)
+  common.getHMIConnection():ExpectNotification("SDL.OnAppPermissionChanged", {
+    appID = common.getHMIAppId(),
+    requestType = { "PROPRIETARY", "OEM_SPECIFIC" },
+  })
+  :ValidIf(function(_, data)
+    if data.params.requestSubType then
+      return false, "SDL.OnAppPermissionChanged notification contains unexpected parameter requestSubType"
+    end
+    return true
+  end)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("App registration", common.registerApp)
+
+runner.Title("Test")
+runner.Step("PTU without requestSubType", policyUpdate)
+runner.Step("SystemRequest with requestSubType", common.systemRequest,
```

Comment:
Shouldn't this be `unsuccessSystemRequest`? Based on the Core PR (https://github.com/smartdevicelink/sdl_core/pull/2149/files#diff-7ec0ea8e1e752c736fb05b5f89890fbfR1956), omitting `RequestSubType` from the policy table should result in Core rejecting all requestSubTypes for that app.

Reply:
@jacobkeeler By default SDL has an empty arrays for both ```RequestType``` and ```RequestSubType``` parameters.
In case if one of these parameters is omitted in policy table update then SDL retain the default value.
Since default value is empty array and this value means processing of any request type/sub-type ```SystemRequest``` from mobile app will be processed successfully.

Reply:
Alright, in that case I think there should be test cases where these parameters are omitted from the default policies, testing that any RequestTypes/RequestSubTypes are DISALLOWED in this case.

Reply:
@jacobkeeler There 2 new test scripts were added in e4e1d24 to cover this case. Please check.